java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -3999,6 +4015,8 @@ static DeviceMemoryBufferView getOffsetsBuffer(long viewHandle) {
   }
 
   // Native Methods
+  private static native int distinctCount(long handle, boolean nullsIncluded, boolean nansAreNulls);
```

Comment:
Why do we need this parameter? I don't see any public Java API calling it. If we are not using it then should we just remove it completely from Java code?

Reply:
I added it thinking that there was a java enum that corresponded to it, but never got around to adding in the enum. I can remove it if we want to.

Reply:
Yes please remove it.